test/UnitTests/PlatformServices.Shared.Unit.Tests/netstandard1.0/ns10ThreadOperationsTests.cs

```diff
@@ -35,26 +35,36 @@ public ThreadOperationsTests()
         public void ExecuteShouldStartTheActionOnANewThread()
         {
             int actionThreadID = 0;
-            Action action = () => { actionThreadID = Thread.CurrentThread.ManagedThreadId; };
-
-            Assert.IsTrue(this.asyncOperations.Execute(action, 1000));
+            void action()
+            {
+                actionThreadID = Thread.CurrentThread.ManagedThreadId;
+            }
 
+            CancellationTokenSource tokenSource = new CancellationTokenSource();
+            Assert.IsTrue(this.asyncOperations.Execute(action, 1000, tokenSource.Token));
             Assert.AreNotEqual(Thread.CurrentThread.ManagedThreadId, actionThreadID);
         }
 
         [TestMethod]
         public void ExecuteShouldReturnFalseIftheActionTimesout()
         {
-            Action action = () => { Task.Delay(100).Wait(); };
+            void action()
+            {
+                Task.Delay(100).Wait();
+            }
 
-            Assert.IsFalse(this.asyncOperations.Execute(action, 1));
+            CancellationTokenSource tokenSource = new CancellationTokenSource();
```

Comment:
We should add a test where Execute returns false when the cancellationToken is cancelled.

Reply:
Done. Added.